README.md

```diff
@@ -13,7 +13,7 @@ This tool, henceforth referred to simply as "the runtime", builds upon
 the [virtcontainers](https://github.com/containers/virtcontainers)
 project to provide a high-performance standards-compliant runtime that
 creates hardware-virtualized containers which leverage
-[Intel](https://www.intel.com/)'s VT-x technology.
+[Intel](https://www.intel.com/)'s [VT-x technology](https://www.intel.co.uk/content/www/uk/en/virtualization/virtualization-technology/intel-virtualization-technology.html).
```

Comment:
heh. maybe. I only did the edit in this repo to nudge the CI under test, but did consider if we wanted to push the edit to the main repo (or some derivative of the edit anyhow ;-)